wiki/Help_Centre/en.md

```diff
@@ -72,7 +73,7 @@ A couple of hours at the absolute maximum will clear the status for most people
 
 **There might be an issue with your account's current email address.**
 
-If you remember your account's attached email address, or think you do, try visiting the [email recovery](https://osu.ppy.sh/p/forgot-email) page to file a request to have it changed. Fill in the form as best you can and you'll receive a reply within two days.
+If you remember your account's attached email address, or think you do, try visiting the Email section in [account settings](https://osu.ppy.sh/home/account/edit) page to change your email address. Fill out the necessary fields and click `Update` when you're done.
```

Comment:
I don't think they would need to know their email to change it because signing in is done with a username. However, this seems like a problem because to access that page, you need a verification code which would be sent to the forgotten email...? So I'm not sure if this will actually help.

Reply:
This works if they know which email address is connected to their account and if they have access to their account. If they really really have forgotten which email they were using, they would need to send an email to accounts@ppy.sh afaik, as stated in the following paragraph. Something like an email recovery system doesn't seem to exist anymore, that's why I replaced it.